### ISSUE TITLE
json output fails to render - this was probably supposed to be encode()?

### ISSUE THREAD

**Issue by alanjcastonguay** [vmware/vcd-cli#94]
https://github.com/vmware/vcd-cli/blob/3202927be8ea1a949647e41ae106e486bc2480e2/vcd_cli/utils.py#L180

```
  File "lib/python2.7/site-packages/vcd_cli/utils.py", line 180, in stderr
    message = highlight(str(text, 'utf-8'), lexers.JsonLexer(),
TypeError: str() takes at most 1 argument (2 given)
```

**Comment by dsaojose** [comment]
Also encountering this.

**Comment by alanjcastonguay** [comment]
Looks like this was fixed by commit 3c0d184aaff1bc757938ee8c25893210cec89c05
